backend/migration/utils.py

```diff
@@ -154,20 +158,19 @@ def _migrate_collection(
     # upload_records has been deprecated due to the usage of models.Record; models.Record has been deprecated as a
     # structure for uploading due to a `shard_key` field, and now is used only as a result structure.
     # since shard_keys are not supported in migration, we can safely type ignore here and use Records for uploading
+
     while next_offset is not None:
-        records, next_offset = source_client.scroll(
-            source_collection_name,
-            offset=next_offset,
-            limit=batch_size,
-            with_vectors=True,
+        records, next_offset = tqdm(
+            source_client.scroll(
+                source_collection_name,
+                offset=next_offset,
+                limit=batch_size,
+                with_vectors=True,
+            )
         )
         dest_client.upload_points(destination_collection_name, records, wait=True)
-    source_client_vectors_count = source_client.get_collection(
-        source_collection_name
-    ).vectors_count
-    dest_client_vectors_count = dest_client.get_collection(
-        destination_collection_name
-    ).vectors_count
+    source_client_vectors_count = source_client.count(source_collection_name).count
+    dest_client_vectors_count = dest_client.count(destination_collection_name).count
```

Comment:
I think we should be using `dest_client.get_collection(...).points_count`

Reply:
https://github.com/qdrant/qdrant-client/blob/d18cb1702f4cf8155766c7b32d1e4a68af11cd6a/qdrant_client/qdrant_remote.py#L1272

This returns`count`.

Reply:
I was talking about `.get_collection` and not `.count`

https://github.com/qdrant/qdrant-client/blob/d18cb1702f4cf8155766c7b32d1e4a68af11cd6a/qdrant_client/http/models/models.py#L169

Anyway both should be equivalent, so it is okay

Reply:
yaa..